### PR TITLE
Hotfix/image upload adminui

### PR DESCRIPTION
**Pull request by ra-external**
- remove use of custom wysiwyg field
- also (unrelated to bug) remove podcast and youtube lists

### CODE REVIEW

File: custom-field-wysiwyg/views/Editor.tsx [hunk 1]
```diff
@@ -7,51 +7,46 @@ const editorConfig = {};
...
+	quickbars_selection_toolbar: 'bold italic underline strikethrough | h1 h2 h3 | quicklink blockquote removeformat',
 	quickbars_insert_toolbar: false
 };
 
 const defaultPlugins = 'link lists code autoresize paste quickbars hr table emoticons image media link code imagetools tinydrive';
 
 const defaultToolbar =
 	// 'formatselect forecolor | alignleft aligncenter alignright alignjustify | bullist numlist indent outdent | link unlink | insertfile image table emoticons hr | code';
-	'insertfile image link | code\'';
+	"insertfile image link | code'";
```
Comment (amorote) [inline custom-field-wysiwyg/views/Editor.tsx:25]: Is this **'** after _code_ needed?

Comment (amorote) [inline custom-field-wysiwyg/views/Editor.tsx:25]: Is this ' after _code_ needed?

File: lists/Proposal.ts [hunk 2]
```diff
@@ -1,16 +1,45 @@
...
 export default keystone => {
 	keystone.createList('Proposal', {
 		schemaDoc: 'Proposed Content',
 		labelField: 'title',
 
+		access: {
+			create: ({ authentication }) => {
+				log('Proposal authentication');
+				log(authentication);
+				return !emptyObj(authentication) && authentication.item && authentication.item.isAdmin;
```
Comment (amorote) [inline lists/Proposal.ts:30]: This can be exported to a function if the same code is used 3 times

Comment (ra-external) [reply lists/Proposal.ts:30]: Sí, buena idea.  La próxima ronda

**Pull request merged**